content/docs/accessibility.md

```diff
@@ -65,13 +65,8 @@ function ListItem({ item }) {
 function Glossary(props) {
   return (
     <dl>
-      {props.items.map(item => (
-        // Fragments should also have a `key` prop when mapping collections
-        <Fragment key={item.id}>
-          <dt>{item.term}</dt>
-          <dd>{item.description}</dd>
-        </Fragment>
-      ))}
+      <ListItem item={term: 'Fragment', description: 'A piece of something.' }>
+      <ListItem item={term: 'React', description: 'Respond.' }>
```

Comment:
I think it improves the doc to show the `ListItem` used inside `Glossary`, good idea! 

However it would not be used like this in applications as the reason to componentise the `<dt>`/`<dd>` combination would be to re-use it. Otherwise I personally would just write them inline. Saves code.

Can you change this to use a `map` that then renders the `ListItem` component? THAT is actually he great benefit of `<Fragment>` in these cases.  Then it also flows nicely into the example where you show that the `<Fragment>` also needs a key when being mapped in place.

Reply:
I was thinking of just showing it being used like what was done [here](https://reactjs.org/docs/components-and-props.html#composing-components), but that makes sense. Should be updated now to match your suggestions.